.github/workflows/build.yml

```diff
@@ -16,9 +16,7 @@ on:
   workflow_dispatch: {}
 env:
   PROVIDER: command
-  PULUMI_LOCAL_NUGET: ${{ github.workspace }}/nuget
   TRAVIS_OS_NAME: linux
-  PULUMI_GO_DEP_ROOT: ${{ github.workspace }}/..
```

Comment:
@pose it's being removed here. Not sure why, we must have changed something in ci-mgmt.

We might be able to just add it here is an override for this repo https://github.com/pulumi/pulumi-command/blob/969c18bfddf46c4bded87c5f3ba473756f7f2324/.ci-mgmt.yaml#L9